test/hostbuild/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+corrosion_import_crate(MANIFEST_PATH ${CMAKE_CURRENT_SOURCE_DIR}/Cargo.toml)
+
+set_property(
+    TARGET rust-host-program
+    PROPERTY CORROSION_USE_HOST_BUILD 1
+)
```

Comment:
Maybe this test could actually run the rust-host-program from cmake from a custom command

Reply:
I can do that, although I think that ends up testing more general cmake features than corrosion. How strong do you feel about it? :)